Clarified the requirements (#4438)
umbraco/UmbracoDocs: Fundamentals/Setup/Server-Setup/Load-Balancing/azure-web-apps.md

## Azure Requirements

> * You will need to setup 2 x App service plans with 1 x web app in each - one for the backoffice (Administrative) environment and another for your scalable public-facing environment (Public)

Reviewer: ```suggestion
* You will need to setup 2 x App service plans with 1 x web app in each - one for the backoffice (Administrative) environment and another for your scalable public-facing environment (Public).
```

> * This'll allow for for proper scaling of the (Administrative) and the (Public) web apps.

Reviewer: ```suggestion
* This will allow for the proper scaling of the Administrative and Public web apps.
```

> * You will need to setup 2 x App service plans with 1 x web app in each - one for the backoffice (Administrative) environment and another for your scalable public-facing environment (Public).
> * This will allow for the proper scaling of the Administrative and Public web apps.
> * The App Service plan with the Administrative web app should only be scaled up. The reason for this is that the web app needs to stay as a single instance.
> * The App Service plan with the Public web app can be scaled both out and up.
> * You will need 1 x SQL server that is shared with these 2 web apps

Reviewer: OK. I think we can make this even better and easier to read:

```suggestion
* 2 x App service plans with 1 x web app in each:
  * One for the backoffice (Administrative) environment and another and
  * One for your scalable public-facing environment (Public).
* 1 x SQL server that is shared with these 2 web apps

The setup above will allow for the proper scaling of the Administrative and Public web apps.

The App Service plan with the Administrative web app should only be scaled up. The reason for this is that the web app needs to stay as a single instance.

The App Service plan with the Public web app can be scaled both out and up.

```

> * The app service plan with the (Administrative) web app should never be scaled out but only up, as the web app needs to stay as a single instance.
> * The app service plan with the (Public) web app can be scaled both out and up and having multiple instanced of it is the exact conctept of load balancing.

Reviewer: ```suggestion
* The App Service plan with the Administrative web app should only be scaled up. The reason for this is that the web app needs to stay as a single instance.
* The App Service plan with the Public web app can be scaled both out and up.
```
